internal/fakecgo/trampolines_stubs.s

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2022 The Ebitengine Authors
+
+//go:build darwin
+// +build darwin
```

Comment:
Let's rename the file to `trampolines_stubs_darwin.s` or `trampolines_darwin.s` instead of build tags here